server/db/models/product.js

```diff
@@ -0,0 +1,17 @@
+const Sequelize = require('sequelize');
+const db = require('../db');
+
+// each row is a product
+const Product = db.define('product', {
```

Comment:
Add validators for empty strings and 'null' to eliminate potential bugs when products are added to the database